src/oc_erchef/apps/chef_index/c_src/fast_xs.c

```diff
@@ -0,0 +1,210 @@
+#include <assert.h>
```

Comment:
I'll have to look at the individual commits to find this, but is there anything different in this code than in the old `chef_reindex` module? Also, why vendor that model over here? Does mover now use `chef_index` for the reindex migration or are we further digging ourselves into a mover hole?